pms/demo/pms_reservation.xml

```diff
@@ -1,13 +1,76 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
     <data noupdate="1">
-        <!-- ??? -->
+        <!-- Economic -->
         <record id="pms_reservation_0" model="pms.reservation">
             <field name="partner_id" ref="base.res_partner_address_27" />
             <field name="pricelist_id" ref="product.list0" />
             <field name="room_type_id" ref="pms_room_type_0" />
             <field name="checkin" eval="DateTime.today() + timedelta(8)" />
             <field name="checkout" eval="DateTime.today() + timedelta(9)" />
         </record>
+        <record id="pms_reservation_1" model="pms.reservation">
```

Comment:
This record should not be created: only the reservations of type "out" (out of service) have the field "clousure_reason_id", what's more, we should target this case to control that when a reservation is not of type "out", set to False the cloususe_reason_id field. On the other hand, if you want to create a reservation of type "out", do not indicate the partner_id, it should automatically take the company itself as partner_id